umbral-pre/src/bindings_python.rs

```diff
@@ -59,7 +59,7 @@ fn hash(data: impl AsRef<[u8]>) -> i64 {
     // we just need something fast that minimizes conflicts.
     let digest = Sha256::new().chain(data).finalize();
     let (chunk, _): (GenericArray<u8, U8>, _) = digest.split();
-    let arr: [u8; 8] = chunk.try_into().unwrap();
+    let arr: [u8; 8] = chunk.into();
```

Comment:
Just out of curiosity: is this change related to these GH Actions in some way?

Reply:
I think it's an accidental change. It's possible that `clippy` was updated between the previous CI run and now, and it resulted in a broken linter rule.